service-discovery/README.adoc

```diff
@@ -139,45 +138,35 @@ Before you can bind Kafka instances in {product-long} to applications on OpenShi
-
-The following tutorial shows how to use the RHOAS CLI to perform service binding. In the tutorial, you create an example Quarkus application and connect this to a Kafka instance. link:https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework that is optimized for serverless, cloud, and Kubernetes environments.
-
-When you perform service binding, the Service Binding Operator injects connection parameters as files into the pod for the application. Because the example Quarkus application uses the `quarkus-kubernetes-service-binding` link:https://quarkus.io/guides/deploying-to-kubernetes#service-binding[extension^], the application automatically detects and uses the injected connection parameters.
-
-In general, this automatic injection and detection of connection parameters eliminates the need to manually configure an application to connect to a Kafka instance in {product}. This is a particular advantage if you have many applications in your project that you want to connect to a Kafka instance.
+Also, you can observe that the RHOAS Operator is installed in the `openshift-operators` namespace.
 
 [id="proc-verifying-connection-to-openshift-cluster_{context}"]
-=== Verifying connection to your OpenShift cluster
+== Verifying connection to your OpenShift cluster
 
 [role="_abstract"]
-In this step of the tutorial, you verify that the installed RHOAS Operator is working by using the RHOAS CLI to connect to your OpenShift cluster and retrieve the cluster status.
+After you install the RHOAS Operator, you can verify that the Operator is working by using the RHOAS CLI to connect to your OpenShift cluster and retrieve the cluster status. The following example shows how to verify connection to your OpenShift cluster.
 
 .Prerequisites
 * The RHOAS Operator is installed on your OpenShift cluster. See link:{base-url}{service-binding-url}#proc-installing-rhoas-operator_{context}[Installing the RHOAS Operator on OpenShift].
 * You can access your OpenShift cluster with privileges to create a new project.
-* You've installed the OpenShift CLI. For more information, see link:https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI^].
+* You've installed the OpenShift CLI. For more information, see link:https://docs.openshift.com/container-platform/4.8/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI^].
```

Comment:
OpenShift docs seem to be unstable. It is best to keep external URL refrences as variable (to be fixed later - not this PR).

Reply:
Good call. Will open a separate PR that adds an attribute for the latest OpenShift version, to be used in our URLs.